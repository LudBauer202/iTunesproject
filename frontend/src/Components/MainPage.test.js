import React from "react";
import DisplaySearch from "./MainPage";
import renderer from 'react-test-renderer';

describe("Rendered Page", () => {
    it('Displays', () => {
        const tree = renderer.create(<DisplaySearch/>).toJSON();
        expect(tree).toMatchSnapshot()
    })
})