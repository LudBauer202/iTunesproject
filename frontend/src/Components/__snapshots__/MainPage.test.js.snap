// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Rendered Page Displays 1`] = `
<div
  className="container"
>
  <div
    className="row"
  >
    <div
      className="col"
    >
      <div
        className="HomeSearch"
      >
        <h1
          className="HeaderLogo"
        >
          iTunes
        </h1>
        <input
          onChange={[Function]}
          placeholder="Search any iTunes product"
        />
        <button
          className="btnSearch"
          onClick={[Function]}
        >
          Search
        </button>
        <form>
          <select
            onChange={[Function]}
          >
            <option
              value="all"
            >
              All
            </option>
            <option
              value="movie"
            >
              Movie
            </option>
            <option
              value="podcast"
            >
              Podcast
            </option>
            <option
              value="music"
            >
              Music
            </option>
            <option
              value="musicVideo"
            >
              Music Video
            </option>
            <option
              value="audiobook"
            >
              Audio Book
            </option>
            <option
              value="shortFilm"
            >
              Short Film
            </option>
            <option
              value="tvShow"
            >
              TV Show
            </option>
            <option
              value="software"
            >
              Software
            </option>
            <option
              value="ebook"
            >
              E-book
            </option>
            <option
              value="all"
            >
              All
            </option>
          </select>
        </form>
      </div>
    </div>
  </div>
  <h1
    id="FavHeader"
  >
    Favourites
  </h1>
  <div
    className="row"
  />
</div>
`;
